src/modules/shared/services/notifications-service/notifications.service.ts

```diff
@@ -51,20 +51,21 @@ export abstract class NotificationsService {
 	}
 
 	public static async checkNotifications(): Promise<void> {
-		const mostRecent = NotificationsService.lastFetchedUnreadNotifications?.[0];
-		const lastCheckNotificationTime = mostRecent?.createdAt
-			? parseDatabaseDate(mostRecent?.createdAt).getTime()
-			: 0;
+		const mostRecent = asDate(
+			NotificationsService.lastFetchedUnreadNotifications?.[0].createdAt
```

Comment:
It could be that there are no notifications yet
```suggestion
			NotificationsService.lastFetchedUnreadNotifications?.[0]?.createdAt
```

Reply:
Fixed in a later PR